src/group_theory/subgroup.lean

```diff
@@ -1248,6 +1248,40 @@ by simp only [subset_normal_closure, closure_le]
 le_antisymm (normal_closure_le_normal closure_le_normal_closure)
   (normal_closure_mono subset_closure)
 
+/-- The normal core of a subgroup `H` is the intersection of all of the conjugates of `H`,
+as shown by `subgroup.normal_core_eq_infi`. -/
```

Comment:
Hmm, I think `subgroup.normal_core_eq_infi` is showing something else, namely that is the supremum of all normal subgroups contained in `H`.